spec/ttt_app/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../../lib/ttt_app.rb'
+
+# class TestTicTacToe < Test::Unit::TestCase
+#     def test_printWelcome
+#         assert_equal 'Welcome to Tic Tac Toe Game', TicTacToe.printWelcome, "TicTacToe.printWelcome should return a string  'Welcome to Tic Tac Toe Game'"
+#     end
+# end
```

Comment:
Feel free to remove any commented out code and leave it to Git to track any outdated changes.

Reply:
Alright.